resources/logback.xml

```diff
@@ -0,0 +1,37 @@
+<configuration>
+  <!--
```

Comment:
Dit stukje tekst klopt niet; als deze in `resources` staat komt de config gewoon in de uberjar. Dit is dus juist de default "production" config.